detections/network/inactive_account_have_activity_detected.yml

```diff
@@ -0,0 +1,80 @@
+name: Inactive Account Have Activity Detected
+id: e08aa2f6-7d90-4ab8-af11-da4df38bb7ff
+version: 1
+date: '2024-10-14'
+author: Zaki Zarkasih Al Mustafa
+data_sources:
+- Windows Security 4624
+- Windows Security 4625
+type: Correlation
```

Comment:
We use `type: Correlation` a bit differently in ESCU and this search can perhaps be more of a Hunting/Anomaly  search. Can you update this type to Hunting or Anomaly